clients/fides-js/src/components/notices/NoticeOverlay.tsx

```diff
@@ -1,33 +1,49 @@
-import { h, Fragment, FunctionComponent } from "preact";
+import "../fides.css";
+
+import { Fragment, FunctionComponent, h } from "preact";
 import { useCallback, useMemo, useState } from "preact/hooks";
+
+import { getConsentContext } from "../../lib/consent-context";
 import {
   ConsentMechanism,
   ConsentMethod,
   FidesCookie,
   PrivacyNotice,
+  PrivacyNoticeTranslation,
+  PrivacyNoticeWithPreference,
   SaveConsentPreference,
   ServingComponent,
 } from "../../lib/consent-types";
-import ConsentBanner from "../ConsentBanner";
-
-import { updateConsentPreferences } from "../../lib/preferences";
-import { debugLog } from "../../lib/consent-utils";
-
-import "../fides.css";
-import Overlay from "../Overlay";
-import { NoticeConsentButtons } from "../ConsentButtons";
-import NoticeToggles from "./NoticeToggles";
-import { OverlayProps } from "../types";
-import { useConsentServed } from "../../lib/hooks";
+import { debugLog, getGpcStatusFromNotice } from "../../lib/consent-utils";
+import { resolveConsentValue } from "../../lib/consent-value";
 import {
   getFidesConsentCookie,
   updateCookieFromNoticePreferences,
 } from "../../lib/cookie";
-import PrivacyPolicyLink from "../PrivacyPolicyLink";
 import { dispatchFidesEvent } from "../../lib/events";
-import { resolveConsentValue } from "../../lib/consent-value";
-import { getConsentContext } from "../../lib/consent-context";
+import { useConsentServed } from "../../lib/hooks";
+import {
+  selectBestExperienceConfigTranslation,
+  selectBestNoticeTranslation,
+} from "../../lib/i18n";
+import { updateConsentPreferences } from "../../lib/preferences";
 import { transformConsentToFidesUserPreference } from "../../lib/shared-consent-utils";
+import ConsentBanner from "../ConsentBanner";
+import { NoticeConsentButtons } from "../ConsentButtons";
+import Overlay from "../Overlay";
+import PrivacyPolicyLink from "../PrivacyPolicyLink";
+import { OverlayProps } from "../types";
+import { NoticeToggleProps, NoticeToggles } from "./NoticeToggles";
+
```

Comment:
nit: update the import paths to be absolute `~`

Reply:
I'd definitely like to do that but I don't remember if there's a quirk in the FidesJS codebase for why we don't do this _anywhere_.